### ISSUE TITLE
invalid windows filename in repo

### ISSUE THREAD

**Issue by alexrayne** [contiki-ng/contiki-ng#2187]
Hallow!
try to checkout release 4.8 on NTFS drive got this filure:
```
aborted: file name contains '?', this is reserved name on Windows: b'doc/project/Where-to-report-issues,-ask-questions,-etc.?.md'
```
i expects same problems with fat32/exFAT

**Comment by alexrayne** [comment]
same problem with 
`doc/tutorials/Cooja:-simulating-a-border-router.md `

**Comment by alexrayne** [comment]
that was fixed in adcd7eaea2e9
